app/main.py

```diff
@@ -14,4 +14,11 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    count = 0
+
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
Can you try to solve this task in one line?

Reply:
Hello! I already solve this task! The file at "Files changed". Can you please check it?